python/ola/TestUtils.py

```diff
@@ -32,3 +34,13 @@ def allHashNotEqual(testCase, t):
   h = map(hash, t)
   for pair in itertools.combinations(h, 2):
     testCase.assertNotEqual(pair[0], pair[1])
+
+
+def handleRPCByteOrder(expected):
+  # The RPC header (version and size) is encoded in native format, so flip that
+  # part of the expected data where necessary (as our expected is from a
+  # little endian source
+  if sys.byteorder == 'big':
```

Comment:
Technically we could just run it all of the time, as it wouldn't change the order on an LE machine, but that seemed a bit unnecessary